source/api/presentation/3.0/index.md

```diff
@@ -531,27 +531,48 @@ The value _MUST_ be a string.
 
 A set of user experience features that the publisher of the content would prefer the client to use when presenting the resource. This specification defines the values in the table below. Others may be defined externally as an [extension][prezi30-ldce].
 
+In order to determine the behaviors that are governing a particular resource, there are four inheritance rules:
+* Collections inherit behaviors from their parent Collection.
+* Manifests **DO NOT** inherit behaviors from any parent Collections.
+* Canvases inherit behaviors from their parent Manifest, but **DO NOT** inherit behaviors from any parent Ranges.
+* Ranges inherit behaviors from their parent Range and parent Manifest.
+
+When interpreting the Layout Behaviors and `auto-advance` for Ranges, they govern the rendering of the resources when that particular Range is selected but are not always applied.  A Range with the `behavior` value `continuous`, in a Manifest with the `behavior` value `paged`, would mean that the Manifest's Canvases should be rendered in a paged fashion, unless the range is selected to be viewed, and its included Canvases would be rendered in that context only as being virtually stitched together. This might occur, for example, when a physical scroll is cut into pages and bound into a codex with other pages, and the publisher would like to provide the user the experience of the scroll in its original form.
```

Comment:
I find the first sentence very hard to scan. Not sure this version is any better...

> Clients should interpret behaviors on a Range only when that Range is selected or is in some other way the context for the user's current interaction with the resources. At other times, behaviors on a Range may not apply to the current interaction because the user isn't interacting with the resources in the context of that Range.
> A Range with the `behavior` value `continuous`, in a Manifest with the `behavior` value `paged`, means that the Manifest's Canvases should be rendered in a paged fashion, _unless that Range is selected to be viewed_. Its included Canvases would be rendered as being virtually stitched together in that context only. This might occur, for example, when a physical scroll is cut into pages and bound into a codex with other pages, and the publisher would like to provide the user the experience of the scroll in its original form. 


This whole bit of spec introduces the notion of state history - how did the user come to be looking at resource X? Which is maybe unavoidable but makes me uneasy.

Reply:
I think splitting the first sentence as above helps with clarity.

Reply:
I think it's unavoidable.  Agree the first sentence was cumbersome.  Not sure the second proposed sentence is useful though, as if Canvases don't inherit behaviors from Ranges, so if you're not interacting with a Range, there are no resources that have inherited behavior.

Replace first sentence with above first sentence.